src/components/actionBar/_utils/actionIconSchema.tsx

```diff
@@ -1,9 +1,8 @@
 import { FaExclamationCircle, FaPlusCircle } from "react-icons/fa";
 import { IconType } from "react-icons";
 import React from "react";
-import { SubmissionForm } from "../../modal/submission";
-import { ReportingForm } from "../../modal/reporting";
-import { ActiveIndexStateMethods } from "../../_utils/useActiveIndexState";
+import { Reporting } from "../../modal/Reporting";
+import { Submission } from "../../modal/Submission";
```

Comment:
Change these file names/dir structure to scope them out of the general modal components?

Reply:
Created seperate dirs for usages